declarations/validator.d.ts

```diff
@@ -6,6 +6,7 @@ declare module 'validator' {
   ): boolean;
   export function equals(str: string, comparison: string): boolean;
   export function isAfter(str: string, date?: string): boolean;
+  export function isAfter(str: string, options?: import('../src/options').IsAfterOptions): boolean; // overload
```

Comment:
Interesting. Whoever made this change to validator didn't think of porting it to `isBefore` 😛